scripts/snap/impl/snapToJbMain.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$1" ]; then
+echo "Specify the snapping subfolder. For example: ./snapToJbMain.sh androidx/compose-ui/1.6.0-alpha02 compose ':(exclude)compose/material3'"
+exit 1
+fi
+
+DIR=$(dirname "$0")
+ALL_FOLDERS=${@:1}
+CURRENT_COMMIT=$(git rev-parse --short @)
+BRANCH_TO_RESTORE_IN_THE_END=$(git branch --show-current)
+
+
+TO_JB_MAIN_BRANCH=integration-snap/$CURRENT_COMMIT/to-jb-main
+git checkout --quiet $(git merge-base $CURRENT_COMMIT origin/jb-main) -B $TO_JB_MAIN_BRANCH
```

Comment:
Doesn't work for my case: remote can have not "origin" name

Reply:
removed. local branches should be up-to-date to this point anyway.

Reply:
> local branches should be up-to-date to this point anyway.

Actually, I am wrong, all work can be done in "integration" without updating "jb-main".

Fixed reading the associated remote.